pyproject.toml

```diff
@@ -42,13 +42,13 @@ ignore_missing_imports = true
 
 [tool.poetry]
 name = "spark8t"
-version = "0.0.3"
+version = "0.0.4"
 description = "This project allow you to create a SNAP for interacting with a K8s cluster to submit Spark jobs or run spark shells interactively"
 authors = [
-    "Abhishek Verma <abhishek.verma@canonical.com>",
```

Comment:
I don't think you should remove this line, just the email address.

Reply:
Yes, indeed. I have discussed this with others DP managers, and I believe we should just refer to 

```
 "Canonical Data Platform <data-platform@lists.launchpad.net>"
```